app/controllers/flow_base_controller.rb

```diff
@@ -1,4 +1,4 @@
-class BaseController < ApplicationController
+class FlowBaseController < ApplicationController
```

Comment:
It was conflicting with Blazer's BaseController 
https://github.com/ankane/blazer/blob/master/app/controllers/blazer/base_controller.rb